src/backend/feed/processor.js

```diff
@@ -108,9 +108,11 @@ module.exports = async function processor(job) {
           break;
         case 404:
           logger.warn(`${info.status} Feed not found: ${feed.url}`);
+          feed.saveInvalid(`${info.status} Feed not found: ${feed.url}`);
```

Comment:
I don't think we want to set feeds invalid in here.  Instead, you should do it when the feed queue `.on('fail')` event occurs, since this will give the feed queue a chance to retry the job (e.g., could be a temporary 404).

Reply:
Any issues with line 159 and 163 and the use of ```feed.setInvalid()``` there?

Reply:
I think we only want to invalidate a feed if it comes via the fail event in the queue.